### PR TITLE
Lesson about Module in Python

### PR DESCRIPTION
**Pull request by Pyrotheum1702**

### CODE REVIEW

File: 13-python-modules/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - `pip install` + package name
+# Virtual Environment
+1. Lesson Content:
+  - Virtual environment
+  - Conda
+2. What i learned:
+  - Virtual environments are isolated environments that use to keep dependencies for different projects separated.
+    - For instance: project A need Django ver 4.0, project B need Django ver 4.1, we could make that possible by create 2 different virtual environment for them.
+  - By default, every project on your system will share the same site packages, except for projects in virtual environments which those projects are isolated, separated and have their own packages that also not share with projects anywhere else.
+  - Conda:
```
[the remark this answers is not in the document]
Comment (Pyrotheum1702) [reply 13-python-modules/README.md:49]: idk, try to use conda? not sure about what am gonna i use it for. And then i have to learn how to use it, i don't think it's worth learning about it at the momment.

File: 13-python-modules/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - Import a module as different name:
+    - You can rename the module while importing it using the keyword `as`.
+    - Syntax: `import` + module name + `as` + name
+  - Built-in modules:
+    - Python built-in module could be found here: https://docs.python.org/3/library/
+# Package Manager
+1. Lesson Content:
+  - PIP
+2. What i learned:
+  - What is PIP:
```
[the remark this answers is not in the document]
Comment (Pyrotheum1702) [reply 13-python-modules/README.md:30]: yes, i did try to install a package with it, what else should i try to do with it?

File: 13-python-modules/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  - If you do not have PIP installed, you can find it here: https://pypi.org/project/pip/
+  - You can find python packages from here: https://pypi.org/
+  - To install a package use this syntax:
+    - `pip install` + package name
+# Virtual Environment
+1. Lesson Content:
+  - Virtual environment
+  - Conda
+2. What i learned:
+  - Virtual environments are isolated environments that use to keep dependencies for different projects separated.
```
[the remark this answers is not in the document]
Comment (Pyrotheum1702) [reply 13-python-modules/README.md:46]: yes, how am i supose to show that i did?

File: 13-python-modules/README.md [hunk 4]
```diff
@@ -48,4 +49,30 @@
...
+  - To create a new project using pipenv use: `$ py -m pipenv --python ` + python version. Example: `3.11.2`
+    - Notice: the virtual environment subfolder is not in the project folder that's because pipenv manages virtual environments for projects by keeping them in subdirectory of your user profile directory. In my case i could find it in `D:\Users\Administrator\.virtualenvs`
+  - To install a package using pipenv use: `$ pipenv install ` + package name.
+  - Pipfile:
+    - Is a plain text file contain information about where pipenv get packages from for the project and which packages and what version to use, what version of Python is needed.
+    - Don't edit this file directly, use `pipenv` from the command line to make any changes.
+  - Pipfile.lock:
+    - This file describe all the versions of all the packages were installed in the project including the hashes generated from them, this way we can ensure to both ourselves and others that we are in fact installed the same identical versions.
+3. Personal thought:
+  - Pipenv is just a combination of pip and virtual environment.
```
Comment (vanhiepdam) [inline 13-python-modules/README.md]: one more thing that pipenv has solved pip problem is that

problem: for each package dependencies, it mostly has sub-dependencies and the version of the sub-deps are not managed by pip but pipenv do

sometimes, the sub-deps are updated that makes the parent packages are broken, so managing the version of sub-deps is necessary

**Review by vanhiepdam — APPROVED** [review]

**Pull request merged**